tools/validate/validate.cpp

```diff
@@ -244,6 +251,8 @@ int main() {
 
         if (binary_search(skipped_extensions.begin(), skipped_extensions.end(), extension)) {
             continue;
+        } else if (binary_search(bad_extensions.begin(), bad_extensions.end(), extension)) {

```

Comment:
Eww, else-after-continue. (Not worth resetting testing if we don't need to change anything else.)
```suggestion
        }
        
        if (binary_search(bad_extensions.begin(), bad_extensions.end(), extension)) {
```